lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -776,6 +796,103 @@
             }
           }
 
+          .xseries-action{
+            .xseries-msg{
+              @include float(left);
+              width: flex-grid(9, 12);
+            }
+
+            p+p{
+              margin-top: 0;
+            }
+
+            .message-copy{
+              font-weight: 500;
+            }
+
+            .message-copy-bold{
+              font-weight: 600;
+            }
+
+            .xseries-border-btn {
```

Comment:
Is there not an existing button from the dashboard we can use or extend here? This seems like a lot of duplicated sass otherwise. If this would be a substantial change we can push this cleanup off a bit, as I know button styling isn't cleanly done in the LMS at the moment. Regardless it was worth noting.